components/eamxx/tests/uncoupled/ml_correction/ml_correction_standalone.cpp

```diff
@@ -62,6 +62,7 @@ TEST_CASE("ml_correction-stand-alone", "") {
   reference += 0.1;
   Real reference2 = qv(1, 30);
   reference2 += 0.1;
+  ekat::disable_all_fpes();  // required for importing numpy
```

Comment:
Yes, that is the plan. I didn't do it here because there's no more computation after coming back from python in this test. For consistency, I will re-enable it so that we don't forget to do that in the real ML correction module.